src/cts/zcl_abapgit_cts_api.clas.abap

```diff
@@ -374,13 +375,14 @@ CLASS zcl_abapgit_cts_api IMPLEMENTATION.
 
     CALL FUNCTION 'TRINT_OBJECTS_CHECK_AND_INSERT'
       EXPORTING
-        iv_order       = iv_transport
-        iv_with_dialog = lv_with_dialog
+        iv_order           = iv_transport
+        iv_with_dialog     = lv_with_dialog
+        iv_append_to_order = iv_append_to_order
```

Comment:
hmm, what does `iv_append_to_order` do? I would assume the function module to always adds/appends stuff

Reply:
I tried to make the function add the entries to a task but that seems to work only for customizing transports. With development transports and the append flag, the function adds the entries to the transport order, not the task. 

I will try some more...